x-pack/platform/plugins/shared/security/public/management/role_combo_box/role_combo_box.tsx

```diff
@@ -146,7 +146,9 @@ function renderOption(option: Option) {
       <EuiFlexItem>
         {option.value?.description ? (
           <EuiToolTip position="left" content={option.value?.description}>
-            <EuiText size="s">{option.label}</EuiText>
+            <EuiText size="s" tabIndex={0}>
```

Comment:
I'm unsure about this one: these are options in the combo box, the arrow keys are used to navigate through them. When I tab, it just skips over it.

Is there a different attribute that could be added?

Reply:
@kc13greiner  Yeah... you've found another case of unusual component usage =) I'll take a look at the options how we can deal with similar cases tomorrow morning.